src/NodeSlot.ts

```diff
@@ -139,18 +139,16 @@ export abstract class NodeSlot implements INodeSlot {
 
   draw(
     ctx: CanvasRenderingContext2D,
-    options: IDrawOptions,
-  ) {
-    const {
+    {
       pos,
       colorContext,
       labelColor = "#AAA",
       labelPosition = LabelPosition.Right,
       lowQuality = false,
       highlight = false,
-    } = options
-    let { doStroke = false } = options
-
+      doStroke = false,
+    }: IDrawOptions,
```

Comment:
I think we should avoid unpack of kwargs in function signature, as the default value will be assigned when function is declared versus evaluated at runtime. This should not make any difference for literal defaults, but will make a difference if the ref changes. For example when default color is `LiteGraph.XXX`, which need to be re-evaluated at each func call.